### ISSUE TITLE
Please add how to output the final result in manual

### ISSUE THREAD

**Issue by OnlyBelter** [rcastelo/GSVA#30]
Since I am not familiar with R and `ExpressionSet` class, it's really hart to find the final result, i.e. enrichment score.

Could you add this code into manual explicitly:

```
result <- gsva(...)
result_enrichment_score <- result@assayData$exprs
```

**Comment by rcastelo** [comment]
hi, if you are getting an `ExpressionSet` object as output of the `gsva()` function is because you gave an `ExpressionSet` object as input. If you decide to have your molecular and phenotype data in an `ExpressionSet` object, then you should make an effort to understand how to access the information in such type of object. More concretely, you should go to the [Biobase](https://bioconductor.org/packages/Biobase) package landing page and read through the introduction of the vignette "An introduction to Biobase and ExpressionSets". In its section "5.1. Accessing Data Elements" you will see that there's an accessor method called `exprs()` to extract the molecular data from an `ExpressionSet` object. In general, you should never be obliged to access a S4 slot using the `@` operator and if you feel you do, then probably you should look up for documentation about that S4 class of objects.

In summary, the manual does not explain how to use an `ExpressionSet` object because it assumes that if you are using such a type of S4 object is because you are familiar with it. In the vignette of GSVA, however, you will find some places where this type of object is used, for instance in the section entitled "GSVA for RNA-Seq data".

**Comment by OnlyBelter** [comment]
I got it, thank you!
